CHANGELOG

```diff
@@ -1,3 +1,12 @@
+Release 109 (2019-10-22)
+------------------------
+Add com.snowplowanalytics.snowplow/event_fingerprint_config/jsonschema/1-0-1 (#1009)
+Add com.snowplowanalytics.snowplow/anon_ip/jsonschema/1-0-1 (#1008)
+Patch com.snowplowanalytics.snowplow/atomic/jsonschema/1-0-0 (#906)
+Patch com.mailgun/message_opened/jsonschema/1-0-0 (#1010)
```

Comment:
I don't see this issue in the release milestone. I can add it but I don't know if this is because it's for another release?

Reply:
Thanks! Added it last moment.